### ISSUE TITLE
Heavy survivor mask. Error in materials?

### ISSUE THREAD

**Issue by IdleSol** [CleverRaven/Cataclysm-DDA#71700]
### Describe the bug

The description of the mask says it's made of "low carbon steel". If you look at the components, the only metal item is a "small metal sheet"

The material of the small metal sheet: steel.

### Attach save file

N/A

### Steps to reproduce

N/A

### Expected behavior

N/A

### Versions and configuration

N/A

### Additional context

heavy survivor mask - https://cdda-guide.nornagon.net/item/mask_hsurvivor
small metal sheet - https://cdda-guide.nornagon.net/item/sheet_metal_small

https://github.com/CleverRaven/Cataclysm-DDA/blob/eb9cacd70e0a2b6ac996e9cf2d316c6f3ad339fd/data/json/items/tool_armor.json#L2198

```    "material": [ "leather_treated", "hard_plastic_transp", "lc_steel" ], ```

https://github.com/CleverRaven/Cataclysm-DDA/blob/eb9cacd70e0a2b6ac996e9cf2d316c6f3ad339fd/data/json/recipes/armor/head.json#L1712
```
    "components": [
      [ [ "glasses_bal", 2 ], [ "rigid_plastic_sheet", 1 ], [ "survivor_goggles", 1 ] ],
      [ [ "mask_gas", 1 ], [ "mask_bunker", 1 ], [ "mask_gas_half", 1 ], [ "mask_gas_xs", 1 ], [ "mask_gas_xl", 1 ] ],
      [ [ "sheet_metal_small", 2 ] ],
      [ [ "wax_any", 1, "LIST" ], [ "any_tallow", 4, "LIST" ] ]
    ],
```

https://github.com/CleverRaven/Cataclysm-DDA/blob/eb9cacd70e0a2b6ac996e9cf2d316c6f3ad339fd/data/json/items/resources/metal.json#L240

```    "material": [ "steel" ],```

**Comment by Kantonine** [comment]
> ### Describe the bug
> 
> The description of the mask says it's made of "low carbon steel". If you look at the components, the only metal item is a "small metal sheet"
> 
> The material of the small metal sheet: steel.
> ### Attach save file
> 
> N/A
> ### Steps to reproduce
> 
> N/A
> ### Expected behavior
> 
> N/A
> ### Screenshots
> 
> _No response_
> ### Versions and configuration
> 
> N/A
> ### Additional context
> 
> heavy survivor mask - https://cdda-guide.nornagon.net/item/mask_hsurvivor small metal sheet - https://cdda-guide.nornagon.net/item/sheet_metal_small
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/eb9cacd70e0a2b6ac996e9cf2d316c6f3ad339fd/data/json/items/tool_armor.json#L2198
> 
> ` "material": [ "leather_treated", "hard_plastic_transp", "lc_steel" ],`
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/eb9cacd70e0a2b6ac996e9cf2d316c6f3ad339fd/data/json/recipes/armor/head.json#L1712
> 
> ```
>     "components": [
>       [ [ "glasses_bal", 2 ], [ "rigid_plastic_sheet", 1 ], [ "survivor_goggles", 1 ] ],
>       [ [ "mask_gas", 1 ], [ "mask_bunker", 1 ], [ "mask_gas_half", 1 ], [ "mask_gas_xs", 1 ], [ "mask_gas_xl", 1 ] ],
>       [ [ "sheet_metal_small", 2 ] ],
>       [ [ "wax_any", 1, "LIST" ], [ "any_tallow", 4, "LIST" ] ]
>     ],
> ```
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/eb9cacd70e0a2b6ac996e9cf2d316c6f3ad339fd/data/json/items/resources/metal.json#L240
> 
> ` "material": [ "steel" ],`

id say this is intentional for the mask to have lc steel, as the armored jackets and vests use low carbon steel even though they use mtal sheets, same with the sheet metal armor pieces

**Comment by fairyarmadillo** [comment]
As the person who PR'd it, it is intentional that the mask has low carbon steel.

This is not an error, it's just that sheet metal currently isn't differentiated.

**Comment by IdleSol** [comment]
Use it if you want:
```
  {
    "id": "sheet_lc_steel_small",
    "type": "AMMO",
    "copy-from": "sheet_metal_small",
    "name": { "str": "small mild steel sheet", "str_pl": "small mild steel sheets" },
    "description": "A small sheet of mild steel.",
    "material": [ "lc_steel" ]
  },
```
```
   {
    "result": "sheet_lc_steel_small",
    "type": "recipe",
    "activity_level": "MODERATE_EXERCISE",
    "category": "CC_OTHER",
    "subcategory": "CSC_OTHER_PARTS",
    "skill_used": "fabrication",
    "difficulty": 5,
    "time": "180 m",
    "batch_time_factors": [ 90, 2 ],
    "book_learn": [
      [ "reference_fabrication1", 5 ],
      [ "welding_book", 5 ],
      [ "alloy_book", 5 ]
    ],
    "proficiencies": [
      { "proficiency": "prof_metalworking" },
      { "proficiency": "prof_blacksmithing" }
    ],
    "using": [ [ "blacksmithing_standard", 1 ] ],
    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ] ],
    "components": [ [ [ "lc_steel_chunk", 1 ] ] ]
  },
```

It seems to be working. The problem is that you can only learn the recipe from books. Accordingly, without books it will be impossible to create a mask.

There are two options:
1. Leave it as it is
2. Change the material of the mask to steel. And add an advanced mask at level 7-8, where you can use the new material. And at the same time, a device for night and infrared vision. You're working on them, right?

If anyone will take on adding a new sheet. Then pay attention to the armored vests, too. It's the same problem.

**Comment by Kantonine** [comment]
The materials for the survivor items are all over the place anyways tbqh, the Survivor Boots and Winter Survivor Boots use generic Steel, the Steel-Plated Kevlar Jumpsuit Uses Tempered Steel, the Heavy Mask uses Low carbon Steel, and The Gloves havent been touched at all yet.
as for the Armored Jackets, I personally dont think it would be good for balance to make them uncraftable without finding books first *just* to make the non case hardened ones, its not exactly rocket science to be able to sew pre-drilled sheet metal onto a jacket, and this also makes sheet metal armor super unaccessable too

**Comment by IdleSol** [comment]
> the Steel-Plated Kevlar Jumpsuit Uses Tempered Steel,

And it's just as dependent on books or luck to find the right scraps of steel.
https://github.com/CleverRaven/Cataclysm-DDA/blob/c4fdcbe8a22fd02a8f12f77115b576b0acbbf8c8/data/json/recipes/armor/suit.json#L1801

But it is the mild steel that is needed for mending. Yes, it can be repaired with plastic and leather. Because the game does not take into account whether it is leather, plastic or steel that is broken.

And we're also lucky that we can't take it apart and get a different type of steel. Or cut it.

I'm not strictly in favor of realism. But turning one material into another, without any justification. Isn't that a bit much?

Third option, add tools and materials to change the type of steel during manufacturing. That's what they did with the hardened armored jeans.
https://cdda-guide.nornagon.net/item/jeans_mod_ch

@fairyarmadillo 
There's also a problem with the mask. Cut it.
![test](https://github.com/CleverRaven/Cataclysm-DDA/assets/155764584/431de399-d52c-434b-b94b-cfffb4909dbd)
There are problems with the rest of the masks too.

**Comment by fairyarmadillo** [comment]
> The materials for the survivor items are all over the place anyways tbqh, the Survivor Boots and Winter Survivor Boots use generic Steel, the Steel-Plated Kevlar Jumpsuit Uses Tempered Steel, the Heavy Mask uses Low carbon Steel, and The Gloves havent been touched at all yet. as for the Armored Jackets, I personally dont think it would be good for balance to make them uncraftable without finding books first _just_ to make the non case hardened ones, its not exactly rocket science to be able to sew pre-drilled sheet metal onto a jacket, and this also makes sheet metal armor super unaccessable too

I am working through all survivor gear to update the items as I did in #70666 . Further updates are waiting on #71584 which will create a need for homemade acid/bileproof equipment. Survivor gear is generally being retooled to be handcrafted mutation-friendly stuff made without complicated forging processes, and is designed for dealing with most of the hazards that zombies can dish out. It will lack the high-tech features of Hub-01 gear, the melee protection of medieval armor, or the ballistic protection of military gear, but will continue to serve as a sensible everyday set. All survivor gear will use low or mild steel, because it's homemade apocalypse stuff designed to be practical, not the best of the best. This will also make it much easier to repair on the go, which is a primary concern for somebody calling themselves a survivor.

So please just sit tight and let me get to the suits etc. I am working on them, but I want to finish the PR that will make the changes sensible first.

As for the mask, if someone wants to put in a PR to make sheet metal into low steel, that's a separate issue. I'm not sure why the mask is giving synthetic fabric on deconstruction though.

**Comment by Kantonine** [comment]
Thank You for the clarification about sheet metal! I also definitely had the feeling the Survivor suits would become a lot like sunrise suits! though i do have a small question about them, are there any plans to possibly incoporate a kevlar vest into them? i feel any survivor that could reasonably make a kevlar jumpsuit, and also the mercenary body armor as well, wouldnt be difficult to incorporate it into the jumpsuit itself, it feels like a nobrainer. would that become a matter of Balance?

**Comment by fairyarmadillo** [comment]
Yes, the rough plan is:
Light survivor suit - a reinforced nylon jumpsuit with a cut-resistant kevlar lining and an outer coating of something like Flexseal so that it's resistant to acid and other liquids..
Survivor suit - As above, but with a ballistic kevlar vest sewn into the torso.
Heavy survivor suit - As above, but with groin and neck protectors from an ESAPI vest, a steel breastplate, and some more protection on the limbs.

Survivor firesuit - As the survivor suit, but uses a nomex flightsuit as its base instead of a nylon jumpsuit.

The idea is that this stuff is great for getting barfed on and tossed around by zombies, but isn't the best thing to bring to a gunfight and won't shut down a juggernaut as well as tempered plate mail. I'm also going to try to cut down on the cruft and obsolete items that are just like "what if a pair of sunglasses had KEVLAR"

**Comment by Kantonine** [comment]
> Yes, the rough plan is: Light survivor suit - a reinforced nylon jumpsuit with a cut-resistant kevlar lining and an outer coating of something like Flexseal so that it's resistant to acid and other liquids. Survivor suit - As above, but with a ballistic kevlar vest sewn into the torso. Heavy survivor suit - As above, but with groin and neck protectors from an ESAPI vest, a steel breastplate, and some more protection on the limbs.
> 
> Survivor firesuit - As the survivor suit, but uses a nomex flightsuit as its base instead of a nylon jumpsuit.
> 

Thats great! for a long while now ive stayed away from the survivor suits as they were kinda meh for what they were, and only really used the MBA and Survivor Cargos (which i might do some revisions on tbqh, theyre a bit old styled at this point)

>I'm also going to try to cut down on the cruft and obsolete items that are just like "what if a pair of sunglasses had KEVLAR"

Thats also a really good change, it made _zero_ sense to have the goggles or survivor masks to use kevlar over the eyes, and the transfer over to the impact plastic was much better

**Comment by fairyarmadillo** [comment]
> Thats great! for a long while now ive stayed away from the survivor suits as they were kinda meh for what they were, and only really used the MBA and Survivor Cargos (which i might do some revisions on tbqh, theyre a bit old styled at this point)

I am going to do literally every survivor item. I've got a whole thing drawn up, and I'd like them all to have parity with each other so we don't get a bunch of inconsistencies like we do now.

**Comment by Kantonine** [comment]
> > Thats great! for a long while now ive stayed away from the survivor suits as they were kinda meh for what they were, and only really used the MBA and Survivor Cargos (which i might do some revisions on tbqh, theyre a bit old styled at this point)
> 
> I was planning on doing literally every survivor item. I've got a whole thing drawn up, and I'd like them all to have parity with each other so we don't get a bunch of inconsistencies like we do now.
> 
> I can tag you in the draft PR when it goes up if you wanna work together on em.

Please Do!
